src/styles/blocks/icon.scss

```diff
@@ -0,0 +1,28 @@
+.icon {
+  display: block;
+  width: 24px;
+  height: 24px;
+
+  background-size: cover;
+
+  &--phone {
+    background-image: url(../images/Icon-Phone-call.svg);
+  }
+
+  &--close {
+    background-image: url(../images/Icon-Close.svg);
```

Comment:
Use single quotes for URLs in the background-image property for consistency. For example: ```url('../images/Icon-Close.svg')```